src/Core/Config/ConfigCacheLoader.php

```diff
@@ -160,10 +160,6 @@ private function loadLegacyConfig($name)
 	 */
 	private function loadINIConfigFile($filepath)
 	{
-		if (!file_exists($filepath)) {
```

Comment:
This one isn't necessary because:
- it's a private method
- it is called after a `file_exists()` check